src/components/InputTypes/MultiSelect.jsx

```diff
@@ -18,7 +18,7 @@ const MultiSelect = (props) => {
   function renderOptions() {
     return Object.keys(options).map((option) => {
```

Comment:
Honestly we should prob rename the option parameter to something better to differentiate between the key and the value

Reply:
Renamed this to `choices`. Lemme know what you think!